game.js

```diff
@@ -5,23 +5,82 @@ const game = {
   oState: [],
   savedGame: { states: [[], []], xTurn: true },
   gamesCounter: 1,
+  board3x3: true,
```

Comment:
is it possible to add boardSize property and determine the state based only on this property? it look's like you need to hardcode too many things in order to provide this solution.... maybe tomorrow there will be requirement to have an 7*7 board?
the general idea is to find simple solution with minimum effort to support other cases if it will be required...

Reply:
its really making more sense, i didnt think about it.